Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14
```

Comment:
node 14 is ancient -- any chance of upgrading it to current LTS or least 16?

Reply:
I would rather do that in a separate PR because it will probably break the project dependencies, which are very outdated too. I just picked the version that is specified [here](https://github.com/Trillium-Solutions/gtfs-to-html-service/blob/master/package.json#L50).